ansible/roles/os_zabbix/vars/template_os_linux.yml

```diff
@@ -337,3 +337,8 @@ g_template_os_linux:
     url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_yum_update.asciidoc"
     priority: info
     description: "One or more security updates need to be applied to the host"
+
+  - name: "pcp items send failed on {HOST.NAME}"
+    expression: "{Template OS Linux:kernel.uname.sysname.max(#3)}<1"
```

Comment:
I believe uname.sysname is a string, therefore a numeric operation (`max`) won't work on it.

Consider using `nodata` instead.

Also, make sure you test this to make sure it's having the desired effect.